Source/ASDisplayNode.mm

```diff
@@ -574,9 +572,6 @@ - (UIView *)_locked_viewToLoad
     _viewBlock = nil;
     _viewClass = [view class];
   } else {
-    if (!_viewClass) {
-      _viewClass = [self.class viewClass];
-    }
```

Comment:
These are now established during `init` so that the values are invariant across the node's lifetime.